pc2cps

```diff
@@ -9,7 +9,8 @@ import shlex
 import subprocess
 import sys
 
-#==============================================================================
+
+# ==============================================================================
```

Comment:
Please delete the terminal character to keep these at 79 characters.

TBH, I am not a fan of PEP-8. I particularly loath the requirement for two empty lines, because it is very difficult to remember when to use two and when to use only one, and it's also completely unnecessary when using separator lines (which are much better for legibility anyway than just whitespace). I would prefer disabling that particular check.